utils/build/docker/php/apache-mod/php.conf

```diff
@@ -7,6 +7,9 @@
         RewriteRule "^/identify-propagate$" "/identify-propagate/"
         RewriteRule "^/headers$" "/headers/"
         RewriteRule "^/status$" "/status/"
+        RewriteRule "^/custom_event$" "/custom_event/"
```

Comment:
Could you write doc that explain what those endpoint should be doing in `docs/weblog/`?

Reply:
Done